Spigot-API-Patches/0193-Add-brand-support-to-Player.patch

```diff
@@ -0,0 +1,27 @@
+From 825331b7e13353a38f2e934045700321d1a77b68 Mon Sep 17 00:00:00 2001
+From: DigitalRegent <misterwener@gmail.com>
+Date: Mon, 6 Apr 2020 17:15:19 +0200
+Subject: [PATCH] Add brand support to Player
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 877ea4f3..6adc9ce4 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -2063,6 +2063,13 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+         }
+     }
+ 
++    /**
++     * Returns player's client brand name. If the client didn't send this information, the brand name will be null.<br>
++     * For the Notchian client this name defaults to <code>vanilla</code>. Some modified clients report other names such as <code>forge</code>.<br>
++     * @return client brand name
++     */
++    @Nullable String getClientBrandName();
```

Comment:
Put method on next line

Reply:
Fixed.

Reply:
This is fine. Nullable can be on the same line.